docs/unity/part-4.md

```diff
@@ -342,43 +342,42 @@ spacetime generate --lang csharp --out-dir ../client-unity/Assets/autogen
 All that's left is to modify our `PlayerController` on the client to call the `update_player_input` reducer. Open `PlayerController.cs` and add an `Update` function:
 
 ```cs
-    public void Update()
```

Comment:
I fixed formatting issues here (which causes annoying whitespace issues when copy/pasting) and also I removed a reference to `testInputEnabled`/`testInput` which is not used in the tutorial and causes warnings

Reply:
Use this setting on Github to get a cleaner diff

![image](https://github.com/user-attachments/assets/9e7f8c31-e1f2-4739-820b-251540008e2c)